### DISCUSSION TITLE
Boosting Awareness of Our Integrations

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#5515] [Feedback]
Here's a pattern I see on feedback calls: 

- User talks about how they love Parabol and also track all their tasks in Jira 
- I ask if they know we have a Jira integration 
- User says they did not know, they get excited and ask to learn more about it 

It most recently happened with [this user](https://app.hubspot.com/contacts/3888472/contact/28389851) 🔒 , who said:

>  We do everything in Jira and did not know about your Jira integration. I'm going to check that out.

This is a place to track this kind of feedback and discuss how to make folks more aware of our integrations.

Related to [issue 5359](https://github.com/ParabolInc/parabol/issues/5359)

**Comment by BartoszJarocki** [comment]
I've added a few tasks related to this topic which I think should help a bit: 
https://github.com/ParabolInc/parabol/issues/5533 
https://github.com/ParabolInc/parabol/issues/5535
https://github.com/ParabolInc/parabol/issues/5535

**Comment by drewhousman1010** [comment]
A [scrum master](https://app.hubspot.com/contacts/3888472/contact/29920701) 🔒  who has used Parabol extensively across 3 teams said:

> Do you have a Jira integration? We'd need one to justify buying the product.

**Comment by ackernaut** [comment]
We started some exploration [here](https://www.notion.so/parabol/Integration-awareness-9249d35566d24629bce225397703bf9b) and [here](https://www.figma.com/file/uHg5uexCwnZI6UZ9hA4jVB/Integration-Awareness?node-id=0%3A1) including open questions, ideas, and next steps. We can continue to capture comments and feedback on this discussion item in ad hoc fashion, of course!

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/12192201) 🔒 says:

> We use Jira, do you integrate with them? 

For what it's worth, he also automatically assumed the integration was a paid feature after I told him about it.
